types/glob-stream/index.d.ts

```diff
@@ -7,33 +7,42 @@
 
 /// <reference types="node" />
 
-import glob = require('glob');
+import { PicomatchOptions } from "picomatch";
+import { Readable } from "streamx";
 
-declare function GlobStream(glob: string | string[]): NodeJS.ReadableStream;
-declare function GlobStream(glob: string | string[], options: GlobStream.Options): NodeJS.ReadableStream;
+declare function globStream(glob: string | string[]): Readable<globStream.Entry>;
+declare function globStream(glob: string | string[], options: globStream.Options): Readable<globStream.Entry>;
 
-declare namespace GlobStream {
+declare namespace globStream {
     export interface Entry {
         cwd: string;
         base: string;
         path: string;
     }
 
-    export type UniqueByStringPredicate = 'cwd' | 'base' | 'path';
+    export type UniqueByStringPredicate = keyof Entry;
     export type UniqueByFunctionPredicate = (entry: Entry) => string;
 
-    export interface Options extends glob.GlobOptions {
+    export interface Options extends PicomatchOptions {
```

Comment:
Do you mind leaving a comment here regarding why this is pictomatch and not anymatch? e.g.

> `anymatch` passes options down to `picomatch`. We use `pictomatch`'s options here as `anymatch`'s types are hand-written and incorrectly describe the set of possible options that are passed through.

Reply:
Alright! Should be up now
Thanks for taking a look

Reply:
Alright! Should be up now
Thanks for taking a look